dbt/include/spark/macros/materializations/incremental/incremental.sql

```diff
@@ -16,7 +16,7 @@
   {%- set incremental_predicates = config.get('predicates', none) or config.get('incremental_predicates', none) -%}
   {%- set target_relation = this -%}
   {%- set existing_relation = load_relation(this) -%}
-  {%- set tmp_relation = make_temp_relation(this) -%}
+  {% set tmp_relation = this.incorporate(path = {"identifier": this.identifier ~ '__dbt_tmp'}) -%}
```

Comment:
This is vendoring [this](https://github.com/dbt-labs/dbt-spark/blob/1f4442a4dcf9719b1670278760772a07506681fc/dbt/include/spark/macros/adapters.sql#L384-L392) code so that the line omitting the database and schema can be removed. Removing it in the macro itself causes other issues as other macros depend on that change.